modules/redis/5.0.4/overlay/BUILD.bazel

```diff
@@ -0,0 +1,205 @@
+        "src/crc16.c",
+        "src/crc64.c",
+        "src/db.c",
+        "src/debug.c",
+        "src/defrag.c",
+        "src/dict.c",
+        "src/endianconv.c",
+        "src/evict.c",
+        "src/expire.c",
+        "src/geo.c",
+        "src/geohash.c",
+        "src/geohash_helper.c",
+        "src/hyperloglog.c",
+        "src/intset.c",
+        "src/latency.c",
+        "src/lazyfree.c",
+        "src/listpack.c",
+        "src/localtime.c",
+        "src/lolwut.c",
+        "src/lolwut5.c",
+        "src/lzf_c.c",
+        "src/lzf_d.c",
+        "src/memtest.c",
+        "src/module.c",
+        "src/multi.c",
+        "src/networking.c",
+        "src/notify.c",
+        "src/object.c",
+        "src/pqsort.c",
+        "src/pubsub.c",
+        "src/quicklist.c",
+        "src/rand.c",
+        "src/rax.c",
+        "src/rdb.c",
+        "src/redis-check-aof.c",
+        "src/redis-check-rdb.c",
+        "src/release.c",
+        "src/replication.c",
+        "src/rio.c",
+        "src/scripting.c",
+        "src/sds.c",
+        "src/sentinel.c",
+        "src/server.c",
+        "src/setproctitle.c",
+        "src/sha1.c",
+        "src/siphash.c",
+        "src/slowlog.c",
+        "src/sort.c",
+        "src/sparkline.c",
+        "src/syncio.c",
+        "src/t_hash.c",
+        "src/t_list.c",
+        "src/t_set.c",
+        "src/t_stream.c",
+        "src/t_string.c",
+        "src/t_zset.c",
+        "src/util.c",
+        "src/ziplist.c",
+        "src/zipmap.c",
+        "src/zmalloc.c",
+    ] + glob(["src/*.h"]) + ["release.h"],
+    textual_hdrs = glob(["src/ae_*.c"]),
+    includes = ["src"],
+    copts = select({
+        "@platforms//os:linux": ["-fcommon"],
+        "//conditions:default": [],
+    }),
+    deps = [
+        ":hiredis",
+        ":lua",
+    ],
+)
+
+cc_library(
+    name = "redis_cli_lib",
+    srcs = [
+        "src/adlist.c",
+        "src/ae.c",
+        "src/anet.c",
+        "src/crc16.c",
+        "src/crc64.c",
+        "src/dict.c",
+        "src/redis-cli.c",
+        "src/release.c",
+        "src/siphash.c",
+        "src/zmalloc.c",
+    ] + glob(["src/*.h"]) + ["release.h"],
+    textual_hdrs = glob(["src/ae_*.c"]),
+    includes = ["src"],
+    copts = select({
+        "@platforms//os:linux": ["-fcommon"],
+        "//conditions:default": [],
+    }),
+    deps = [
+        ":hiredis",
+        ":linenoise",
+        ":lua",
+    ],
+)
+
+cc_binary(
+    name = "redis-server",
+    deps = [":redis_lib"],
+    visibility = ["//visibility:public"],
+    linkopts = select({
+        "@platforms//os:linux": [
+            "-ldl",
+            "-lm",
+            "-lpthread",
+        ],
+        "@platforms//os:osx": [
+            "-lm",
+        ],
+        "//conditions:default": [],
+    }),
+)
+
+cc_binary(
+    name = "redis-cli",
+    deps = [":redis_cli_lib"],
+    visibility = ["//visibility:public"],
+    linkopts = select({
+        "@platforms//os:linux": [
```

Comment:
this module isn't testing on windows anyway, should we just pass `-lm` unconditionally here?